ReferenceTests/src/tests/examples3d.jl

```diff
@@ -40,15 +40,15 @@ end
     dir_scaled = Vec3f0(
         dir[1] * scene.transformation.scale[][1],
         0.0,
-        dir[3] * scene.transformation.scale[][2],
+        dir[3] * scene.transformation.scale[][3],
     )
     cam.upvector[] = (0.0, 0.0, 1.0)
     cam.lookat[] = minimum(scene_limits(scene)) + dir_scaled
-    cam.eyeposition[] = (cam.lookat[][1], cam.lookat[][2] + 6.3, cam.lookat[][3])
```

Comment:
Ah actually it's probably just because the eyeposition was moved closer here. But I realize now that the orthographic camera should adjust `eyeposition` on zoom...

Reply:
Ok, I've restored the previous setting and changed `zoom_mult` instead.